snappi_ixnetwork/trafficitem.py

```diff
@@ -84,31 +80,48 @@ class TrafficItem(CustomField):
     _PFCPAUSE = {
         "dst": "pfcPause.header.header.dstAddress",
         "src": "pfcPause.header.header.srcAddress",
-        "ether_type": "pfcPause.header.header.ethertype@int",
-        "control_op_code": "pfcPause.header.macControl.controlOpcode@int",
-        "class_enable_vector": "pfcPause.header.macControl.priorityEnableVector@int",
-        "pause_class_0": "pfcPause.header.macControl.pauseQuanta.pfcQueue0@int",
-        "pause_class_1": "pfcPause.header.macControl.pauseQuanta.pfcQueue1@int",
-        "pause_class_2": "pfcPause.header.macControl.pauseQuanta.pfcQueue2@int",
-        "pause_class_3": "pfcPause.header.macControl.pauseQuanta.pfcQueue3@int",
-        "pause_class_4": "pfcPause.header.macControl.pauseQuanta.pfcQueue4@int",
-        "pause_class_5": "pfcPause.header.macControl.pauseQuanta.pfcQueue5@int",
-        "pause_class_6": "pfcPause.header.macControl.pauseQuanta.pfcQueue6@int",
-        "pause_class_7": "pfcPause.header.macControl.pauseQuanta.pfcQueue7@int",
+        "ether_type": "pfcPause.header.header.ethertype",
+        "control_op_code": "pfcPause.header.macControl.controlOpcode",
+        "class_enable_vector": "pfcPause.header.macControl.priorityEnableVector",
+        "pause_class_0": "pfcPause.header.macControl.pauseQuanta.pfcQueue0",
+        "pause_class_1": "pfcPause.header.macControl.pauseQuanta.pfcQueue1",
+        "pause_class_2": "pfcPause.header.macControl.pauseQuanta.pfcQueue2",
+        "pause_class_3": "pfcPause.header.macControl.pauseQuanta.pfcQueue3",
+        "pause_class_4": "pfcPause.header.macControl.pauseQuanta.pfcQueue4",
+        "pause_class_5": "pfcPause.header.macControl.pauseQuanta.pfcQueue5",
+        "pause_class_6": "pfcPause.header.macControl.pauseQuanta.pfcQueue6",
+        "pause_class_7": "pfcPause.header.macControl.pauseQuanta.pfcQueue7",
+        "order": [
```

Comment:
do we have any issue if we are not setting in order

Reply:
yes, if we are not following the order, the values are not getting reflected.